try:
    print(1234)
    print(6543)
    int('a')
except ValueError:
    print('Valor errado')
  